README.md

```diff
@@ -0,0 +1,66 @@
+# **gltf-viewer**
+The GLTF Viewer is a sample application allowing for visualizing GLTF data translated from an OnShape model. It is a Node.JS application that runs as an tab inside an Onshape document. Onshape passes the document context to the viewer, which is used to help generate the GLTF visualization.
```

Comment:
The GLTF Viewer is a sample application for visualizing GLTF data from an Onshape model.

Reply:
Maybe re-read this README carefully and check for a few typos.

Reply:
Updated.